spec/glueby/internal/wallet/tapyrus_core_wallet_adapter_spec.rb

```diff
@@ -0,0 +1,274 @@
+RSpec.shared_examples 'If the wallet is unloaded, it should raise WalletUnloaded error.' do
+  let(:rpc_name) { :listunspent }
+  it 'should raise WalletUnloaded error.' do
+    expect(rpc).to receive(rpc_name)
+                     .and_raise(RuntimeError.new('{"code"=>-18, "message"=>"Requested wallet does not exist or is not loaded"}'))
+    expect { subject }
+      .to raise_error(
+        Glueby::Internal::Wallet::Errors::WalletUnloaded,
+        "The wallet #{wallet_id} is unloaded. You should load before use it."
+      )
+  end
+end
+
+ARBITRARY_WALLET_ID = '00000000000000000000000000000000'
```

Comment:
maybe you expect 32 bytes length but this is 16 bytes.

Reply:
Yes, I will fix it.

Reply:
Fixed at b214dafc76eb6f4afb3d991b86bf768d4b43d9d5